src/Docfx.Build/HostServiceCreator.cs

```diff
@@ -63,7 +63,8 @@ public HostServiceCreator(DocumentBuildContext context)
             {
                 if (e is not DocumentException)
                 {
-                    var message = e is ArgumentException or NullReferenceException
+                    // Log detailed exception messages for specific type of exception.
```

Comment:
Consider for all exception types:
1. Log an error message with `e.Message`
2. Log a verbose message with `e.ToString()`.